src/android/io/branch/BranchSDK.java

```diff
@@ -226,19 +219,11 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
      */
     private void initSession(CallbackContext callbackContext)
     {
-        Log.d(LCAT, "start initSession()");
-
         this.activity = this.cordova.getActivity();
 
         this.instance = Branch.getAutoInstance(this.activity.getApplicationContext());
 
-        if (callbackContext == null) {
-            this.instance.initSession(new SessionListener(), activity.getIntent().getData(), activity);            
-        } else {
-            this.instance.initSession(new SessionListener(callbackContext), activity.getIntent().getData(), activity);
-        }
-
-
+        this.instance.initSession(new SessionListener(callbackContext), activity.getIntent().getData(), activity);
```

Comment:
Yes, it would throw an error that the `callbackContext` parameter has an invalid value `null`.